dev/ssl_darwin.go

```diff
@@ -1,62 +1,69 @@
 package dev
 
 import (
-	"errors"
+	"bytes"
 	"fmt"
-	"io/ioutil"
-	"os"
 	"os/exec"
-
-	"github.com/puma/puma-dev/homedir"
 )
 
+// SupportDir is the platform-specific path that contains puma-dev's generated certs
 const SupportDir = "~/Library/Application Support/io.puma.dev"
 
-func LoginKeyChain() (string, error) {
```

Comment:
Nuking this in favor of `security login-keychain` result. 

#NextLevelTesting: We should be able to install the cert into a puma-dev specific keychain, which would then allow us to initialize it with a password and unlock it from the commandline.

Reply:
💡 might be worth investigating if command-line unlock works locally. but, then you'd have to type your password into terminal, which feels even worse.

Reply:
Long-term strategy might be to crib from https://github.com/docker/docker-credential-helpers to allow `puma-dev` to seamlessly read/write from the macos keychain.